modules/terms/partials/beta.adoc

```diff
@@ -0,0 +1,4 @@
+=== beta
+:term-name: beta
+:hover-text: Features in beta are available for testing and feedback. They are not covered by Redpanda Support and should not be used in production environments. 
```

Comment:
Should you distinguish between public and private beta?

Reply:
I don't think so. We don't publicly document features in private beta. Some have managed to slide out, but there are none right now.